errors/daemon.go

```diff
@@ -385,6 +385,14 @@ var (
 		HTTPStatusCode: http.StatusInternalServerError,
 	})
 
+	// ErrorCodeVolumeName is generated when the name of named volume isn't valid.
+	ErrorCodeVolumeName = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "VOLUMENAME",
+		Message:        "%s includes invalid characters for a local volume name, only %s are allowed",
+		Description:    "The name of volume is invalid",
+		HTTPStatusCode: http.StatusInternalServerError,
```

Comment:
either is fine because at some point (after the errcode disucssions) we'll be looking at each error and making sure it has the right status code.  But getting it to be 400 now is good :-)